src/main/resources/bamm/meta-model/2.0.0/aspect-meta-model-shapes.ttl

```diff
@@ -243,15 +269,16 @@ bamm:PropertyShape
       sh:class bamm:Characteristic ;
       sh:sparql [
          a sh:SPARQLConstraint ;
-         sh:message "A cycle in the Aspect Model was detected" ;
+         sh:message "A cycle in the Aspect Model was detected via '{$this}'." ;
```

Comment:
Not sure, but maybe like this:
"The Aspect Model contains a cycle which includes '{$this}'. Please remove any cycles that do not allow a finite json payload." 
Please clarify with @atextor and me , cycles are permitted if they do not allow indefinite descent (e.g. Characteristic Left/Right included).